test/fixtures/parser/bigquery/interval_function.yml

```diff
@@ -20,33 +20,34 @@ file:
               - expression:
                   interval_expression:
                     keyword: INTERVAL
-                    function:
-                      function_name: cast
-                      start_bracket: (
-                      expression:
-                        function:
-                        - function_name: TIMESTAMP_DIFF
-                        - start_bracket: (
-                        - expression:
-                            column_reference:
-                            - identifier: session_end
-                            - dot: .
-                            - identifier: eventTimestamp
-                        - comma: ','
-                        - expression:
-                            column_reference:
-                            - identifier: session_start
-                            - dot: .
-                            - identifier: eventTimestamp
-                        - comma: ','
-                        - date_part: SECOND
-                        - end_bracket: )
-                        binary_operator: /
-                        literal: '2'
-                      keyword: AS
-                      data_type:
-                        data_type_identifier: int64
-                      end_bracket: )
+                    expression:
```

Comment:
This change looks bigger than it is. The original `cast` `function` element is now nested inside an `expression` element.